pkg/utils/ehpa/ehpa.go

```diff
@@ -0,0 +1,11 @@
+package ehpa
+
```

Comment:
I'd rather not create folder for ehpa.go in utils.

Reply:
> I'd rather not create folder for ehpa.go in utils.

Then it needs distinguish it is EHPA util function, so function name need to be IsEHPAPredictionEnabled and IsEHPACronEnabled

Reply:
these two function is only available for ehpa, so your origin name can show the meaning for the function . However IsEHPAPredictionEnabled and IsEHPACronEnabled is fine to me